app/models/user.rb

```diff
@@ -1,10 +1,17 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+
```

Comment:
- [ ] Good job so far, but when a user sign up he should be directed to the page to confirm the email in your case when a user sign up he has been directed directly to the home page where he can see other users.
- [ ] Also after login or signup a user can't create a post, there is no way or button that allows the user to create a post, kindly fix that and make sure we can add comments and like a post as well 🙏🏾 

<img width="1077" alt="Screenshot 2023-07-27 at 18 45 13" src="https://github.com/lucascomamala/blog-on-rails/assets/74501523/ec8c085d-7e63-4c53-bdbd-6d6c0976e704">

<img width="1182" alt="Screenshot 2023-07-27 at 18 45 27" src="https://github.com/lucascomamala/blog-on-rails/assets/74501523/44b32722-c8c4-4a06-8e04-428379976b90">

- [ ] And when I've tried to see others' posts and like their posts, I got an error, I can't leave a comment and add a like also some of the posts can't be opened, kindly fix that 🙃 😉 

https://github.com/lucascomamala/blog-on-rails/assets/74501523/0a8dc439-7880-41ad-8c63-85d34a66497f

Reply:
I'm unsure what you mean by "the page to confirm the email". When you create a user an email with a confirmation link is sent to the address you used for sign up. You should be able to see that email displayed in the terminal where you started the server. Like this:
![Capture](https://github.com/lucascomamala/blog-on-rails/assets/10102977/b3dbb09f-6f48-4506-97ad-abb40aa1f276)

Reply:
After logging in there's a green button on the top right you can use to create a new post:
![Capture](https://github.com/lucascomamala/blog-on-rails/assets/10102977/40d78d0c-1c76-40f6-999d-2a810fa68a7e)

As for leaving comments, there's a small form at the bottom of the post, you need to be sure you're actually on a specific post:
![Capture](https://github.com/lucascomamala/blog-on-rails/assets/10102977/cf22caf6-8ded-414f-96be-26937dbbe1af)

Reply:
Hi @lucascomamala when I said `the page to confirm the email` it meant that it would be good for the confirmation link to display in the browser and not in the terminal so that after the user signup, he is directed to the email confirmation link in the browser so he can confirm his account and login easily 🙃

Reply:
I see, as that is not explicitly part of the requirements I will leave it as it is; thanks for the suggestion though, I will keep it in mind going forward. Also, I have fixed the issue with the errors and re-submitted the project